lib/hets/errors.rb

```diff
@@ -1,8 +1,19 @@
 module Hets
   module Errors
+    class NotAHetsError < ::StandardError; end
     class HetsError < ::StandardError; end
 
-    class ExecutionError < HetsError; end
+    class ExecutionError < HetsError
+      attr_reader :abort_execution
+
+      def initialize(msg=nil, abort_execution: false)
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.